batch/batch/resource_usage.py

```diff
@@ -1,22 +1,34 @@
 import asyncio
 import errno
+import io
 import logging
 import os
 import shutil
 import struct
-from typing import Optional, Tuple
+from typing import Optional, Tuple, TYPE_CHECKING
 
 import numpy as np
 import pandas as pd
 
 from hailtop.utils import check_shell_output, sleep_and_backoff, time_msecs, time_ns
+from hailtop.aiotools.fs import AsyncFS
+
+if TYPE_CHECKING:
+    from .worker.worker import Container  # pylint: disable=cyclic-import
```

Comment:
We should go to significant lengths to avoid introducing new circularities into our codebase. Let's see if we can avoid it.